manager/integration/tests/test_basic.py

```diff
@@ -640,7 +632,69 @@ def test_backup_block_deletion():  # NOQA
     17. Delete the backup volume
     18. Cleanup the volume
     """
-    pass
+    backupstore_cleanup(client)
+
+    volume = create_and_check_volume(client, volume_name)
+    host_id = get_self_host_id()
+    volume = volume.attach(hostId=host_id)
+    volume = common.wait_for_volume_healthy(client, volume_name)
+
+    data0 = {'pos': 0,
+             'len': 2 * BACKUP_BLOCK_SIZE_IN_MB * Mi,
+             'content': common.generate_random_data(4 * Mi)}
+
+    bv0, backup0, _, _ = create_backup(client, volume_name, data0)
+
+    data1 = {'pos': 0,
+             'len': BACKUP_BLOCK_SIZE_IN_MB * Mi,
+             'content': common.generate_random_data(2 * Mi)}
```

Comment:
Why did you store the Backup block size not as bytes?
Then this call could look like this `common.generate_random_data(1 * BACKUP_BLOCK_SIZE)`

Reply:
fixed